### PR TITLE
Improved customer support

### PR DESCRIPTION
**Pull request by tristanhcole**
**Describe the Pull Request**
- Swaps intercom to use `react-use-intercom` which has hooks and typescript support. 
- Adds the intercom provider to wrap around the app and always initialize - makes it useful to then use intercom anywhere without having to perform boot sequence. 
- Improves and converts to typescript our Page wrapper. 
- Adds sidebar help icon and org settings and invite user on org switcher dropdown.
- Adds `intercom_app_id` to public config file instead of randomly nested in the frontend code.

Screen Shot 2020-08-04 at 10 23 43 am

**Todo**

- [ ] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [ ] Request review from relevant @person or team
- [ ] QA your pull request. This includes running the app, login, testing changes etc.
- [ ] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Intercom/FAQs at bottom of sidebar](https://trello.com/c/aJqofcz8/507-intercom-faqs-at-bottom-of-sidebar)

**Comment by tristanhcole** [comment]
#407 and #285 should be merged in first

File: config_files/public/docker_test_config.ini [hunk 1]
```diff
@@ -69,4 +69,7 @@ host_v2 = https://api.testwyre.com/v2/
 host    = https://api.testwyre.com/v3/
 
 [POLIPAYMENTS]
-host = https://poliapi.apac.paywithpoli.com/api
\ No newline at end of file
+host = https://poliapi.apac.paywithpoli.com/api
+
+[INTERCOM]
+app_id = kowgw7cm
```
Comment (enjeyw) [inline config_files/public/docker_test_config.ini:75]: is this ok to have public @tristanhcole?

Comment (tristanhcole) [reply config_files/public/docker_test_config.ini:75]: Yes

It's like a heap or GA analytics code https://community.intercom.com/t/intercom-app-id-security/966

Comment (tristanhcole) [reply config_files/public/docker_test_config.ini:75]: > LGTM! Assume that app id is ok to be public?

https://github.com/teamsempo/SempoBlockchain/pull/413#discussion_r466784060

**Comment by tristanhcole** [comment]
> LGTM! Assume that app id is ok to be public?

https://github.com/teamsempo/SempoBlockchain/pull/413#discussion_r466784060

File: config_files/public/docker_test_config.ini (hunk 1 above)
Comment (enjeyw) [reply config_files/public/docker_test_config.ini:75]: Greeeat

**Pull request merged**